roles/ovirt-engine-install-packages/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# tasks file for ovirt-engine-install-packages
+- name: yum install engine
+  yum:
+    name: "{{ ovirt_engine_type }}"
+    state: installed
+    update_cache: yes
+  tags:
+    - skip_yum_install_ovirt_engine
+
+- name: yum install packages
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ packages | default([]) }}"
```

Comment:
the `| default([])` is not needed as you have it set to `ovirt-engine-extensions-aaa-ldap`